### DISCUSSION TITLE
Help installing

### DISCUSSION THREAD

**Discussion by irevelle** [lucidrains/big-sleep#57] [Q&A]
I have watched videos but no one really breaks down what to do and I can't find any instructions

**Comment by LtqxWYEG** [comment]
This might help: https://colab.research.google.com/drive/1NZ7qNH_wH5uBpr7NDYbQ0cpGU64zMM6K
It is my version of the Simplified colab notebook that you can find here, with more explanations. You only need a google account. It doesn't even use your hardware, so it's most certainly faster.

**Reply by marisadi** [reply]
Hi I'm wanting to try this project locally and have run the pip install for big sleep and wondering what else i need to do. I currently get the error: 'CUDA must be available in order to use Big Sleep'. What else do I need to do for setup? Sorry if this is a really basic questions.

**Comment by wolfgangmeyers** [comment]
If you're installing on windows make sure to install pytorch manually following directions on this page: https://pytorch.org/get-started/locally/

**Comment by markgauda** [comment]
I had some issues, but it boiled down to not having a 64bit version python installed. I reinstalled version 3.9.13 64bit and it worked great. Make sure you have the CUDA toolkit installed, as of writing this 11.7 is the newest version and it is running this software fine. Hope this helps someone out there.
